examples/ethers-wasm/src/lib.rs

```diff
@@ -1,90 +1,90 @@
-pub mod utils;
+use std::sync::Arc;
+
+use wasm_bindgen::prelude::*;
+use web_sys::console;
 
-use crate::utils::SIMPLECONTRACT_BIN;
 use ethers::{
     contract::abigen,
     prelude::{ContractFactory, LocalWallet, Provider, SignerMiddleware},
-    providers::{Middleware, Ws},
+    providers::Ws,
 };
-use std::sync::Arc;
-use wasm_bindgen::prelude::*;
-use web_sys::console;
+
+use crate::utils::SIMPLECONTRACT_BIN;
+
+pub mod utils;
 
 // When the `wee_alloc` feature is enabled, use `wee_alloc` as the global
 // allocator.
 #[cfg(feature = "wee_alloc")]
 #[global_allocator]
 static ALLOC: wee_alloc::WeeAlloc = wee_alloc::WeeAlloc::INIT;
 
-#[wasm_bindgen]
-extern "C" {
-    fn alert(s: &str);
-}
-
 macro_rules! log {
     ( $( $t:tt )* ) => {
         web_sys::console::log_1(&format!( $( $t )* ).into());
     }
 }
-
 abigen!(
     SimpleContract,
     "./../contract_abi.json",
     event_derives(serde::Deserialize, serde::Serialize)
 );
 
-/// key[0] of ganache with custom seed `ethers-wasm-seed`
-pub const KEY: &str = "817169e55f14ede54f4fd6a4f2ab4209db14aeeb1b9972b3b28f1560af0a061a";
+/// some keys of ganache with custom seed `ethers-wasm-seed`
+pub const KEYS: [&str; 4] = [
+    "817169e55f14ede54f4fd6a4f2ab4209db14aeeb1b9972b3b28f1560af0a061a",
+    "375715b8ced8bd9b7386ba5dc72efa518aa4379d6a4676d3e26d8f5ff5e7469c",
+    "de7c5d8e884fbe9f0915703ff2c123f4cda56f148eb22ca46d47392acf52bcec",
+    "0bd6bf22f84f96b39258a46ac2a7c482d0b8e1c5f8af0c07fa304a8d875158ec",
+];
```

Comment:
FWIW you can also use the [`Mnemonic`](https://github.com/gakonst/ethers-rs/blob/master/ethers-signers/src/wallet/mnemonic.rs#L78-L80) if you want to make this a bit more robust

Reply:
used mnemonic instead.